cmd/gokr-updater/updater.go

```diff
@@ -41,7 +51,28 @@ func main() {
 		log.Fatal(err)
 	}
 	baseUrl.Path = "/"
+	httpClient, foundMatchingCertificate, err := packer.GetTLSHttpClientByTLSFlag(useTLS, baseUrl)
```

Comment:
This function should be moved to internal; don’t import `main` packages such as the packer

Reply:
We probably should move the whole `httpclient.go` to internal?

Reply:
Yes